modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/service/ServerStatisticsUtil.java

```diff
@@ -8,15 +8,12 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
-import org.osgi.service.component.annotations.Component;
-
 /**
  * @author Michael C. Han
  */
-@Component(enabled = false, service = ServerStatisticsHelper.class)
-public class ServerStatisticsHelper {
+public class ServerStatisticsUtil {
```

Comment:
@RenanUmV Since this class is renamed to `*Util`, I think we should move the package.